ext/intl/breakiterator/breakiterator.stub.php

```diff
@@ -64,14 +64,14 @@ public function preceding(int $offset) {}
     public function previous() {}
 
     /** @return bool|null */
-    public function setText(string $text) {}
+    public function setText(string $string) {}
```

Comment:
As we ended up using in a few other places ... maybe stick with `$text` here?

Reply:
Ah yes, I don't even know why I renamed this. Speaking about the discrepancies between the OO and procedural interfaces, I absolutely agree, these are just errors on my part I'll try to review the differences

Reply:
We should probably test that procedural and OO APIs are the same ... somehow. Testing this in stubs is probably not possible because aliases may be in different files (like here).

Reply:
By the way, do you have any recommendation what parameter names to use in case of `intlcal_before()`?

> We should probably test that procedural and OO APIs are the same. ... somehow. Testing this in stubs is probably not possible because aliases may be in different files (like here).

I think it's possible by adding a new option to gen_stubs. If it is passed, I think it's reasonable to expect from users to run the script for a path where all the necessary definitions are available (e.g. for ext/intl: the directories `.`, and `ext`, `ext/intl` equally work well). Probably we could also add a test to run this verification regularly.

I can implement this a bit later if this idea makes sense to you.

Reply:
> By the way, do you have any recommendation what parameter names to use in case of `intlcal_before()`?

I think the current names $calendar + $other are fine there and the OO API should match them.

> > We should probably test that procedural and OO APIs are the same. ... somehow. Testing this in stubs is probably not possible because aliases may be in different files (like here).
> 
> I could imagine adding a new option to gen_stubs. If it is passed, I think it's reasonable to expect from users to run the script for a path where all the necessary definitions are available (e.g. for ext/intl: the directories `.`, and `ext`, `ext/intl` equally work well). Probably we could also add a test to run this verification regularly.
> 
> I can implement this a bit later if this idea makes sense to you.

Hm, I'm not sure about this, it seems like an annoying complication to the current per-file processing. Possibly we can only dump out the aliases and check the rest via reflection? Though I guess that wouldn't work if we also want to compare @param` types.

Reply:
I can't think of any easier way to validate if parameter names of aliases match than using gen_stub. This could be just a one-time effort for php 8, and then we'll have a whole year to find the details out.

Reply:
This one is still using `$string` :)

Reply:
Ah noo, I got "distracted", sorry